### ISSUE TITLE
how to imporve the mAP and decrease the loss of densenet201_yolo?

### ISSUE THREAD

**Issue by JohnTian** [AlexeyAB/darknet#3516]
Hi, @AlexeyAB .

I used **[densenet201_yolo.cfg](https://github.com/AlexeyAB/darknet/blob/master/build/darknet/x64/densenet201_yolo.cfg)** to train my custom objects, **eight** classes. Because in each image, there are a lot of  small or large objects  which are belong to different class. 

I confused how to imporve the **mAP**, and decrease the **loss**. 

- Is it possible to keep the default **steps**, and only change **max_batches**?
- Is there any problem of my **densenet201_yolo** config file which may enfluence **mAP** and **loss** ?

## 1 Config

- Modified **steps** according to **max_batches**.

```bash
 [net]
 batch=64
 subdivisions=8
 height=416
 width=416
 channels=3
 momentum=0.9
 decay=0.0005
 angle=0
 saturation = 1.5
 exposure = 1.5
 hue=.1
 
 learning_rate=0.0001
 #max_batches = 45000
 max_batches = 80000
 policy=steps
 #steps=100,25000,35000
 steps=[177, 44444, 62222]
 scales=10,.1,.1
```

- Modified **classes** and the corresponding **filters**.

```bash
[convolutional]
 size=1
 stride=1
 pad=1
 filters=65    # change from 125 to 65
 activation=linear
 
 
 [region]
 anchors =  1.3221, 1.73145, 3.19275, 4.00944, 5.05587, 8.09892, 9.47112, 4.84053, 11.2364, 10.0071
 bias_match=1
 classes=8 # change from 20 to 8
 coords=4
 num=5
 softmax=1
 jitter=.3
 rescore=1
 
 object_scale=5
 noobject_scale=1
 class_scale=1
 coord_scale=1
 
 absolute=1
 thresh = .6
 random=0
```
## 2 Result

Because only have **4824** images. **valid.txt** keep the same as **train.txt**.

![chart](https://user-images.githubusercontent.com/7224107/60405669-45f45500-9be4-11e9-81ee-a71f39102486.png)
